CMakeLists.txt

```diff
@@ -7,9 +7,14 @@ CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
 
 PROJECT(OpenTomb)
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -msse3")
+IF( ${CMAKE_COMPILER_IS_GNUCC} )
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -msse3")
+ENDIF()
+IF( ${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
```

Comment:
1. combining both ifs - done
2. enclosing ${...} with double quotes - If I understand you correctly:

IF( CMAKE_COMPILER_IS_GNUCC OR  "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" )

Is what you want, but i have to take out the double quotes around CMAKE_CXX_COMPILER_ID because:

" Policy CMP0054 is not set: Only interpret if() arguments as variables or keywords when unquoted."